### DISCUSSION TITLE
Warning custom logs

### DISCUSSION THREAD

**Discussion by tanguy-michardiere** [vercel/community#5996] [Feature Question] [answered]
### Question

Hello, I don't understand how to send custom logs at the "warning" level such as they can be filtered by "Level: Warning" in the runtime logs console.
I tried using `console.warn`, `console.warn` with a message starting with "WARNING: ", `console.log` with a message starting with "WARNING: ".
Thank you

**Comment by swarnava** [comment]
As of now it looks like we treat `console.warn` as stderr. I have passed your feedback internally to look at this.

**Reply by Jooliashi** [reply, marked as answer]
We have added the fix for this, now `console.warn` will show up as `warning`
